src/core/auth.ts

```diff
@@ -49,15 +49,15 @@ export function composeAuthMsg(
   version: string,
   chainId: string,
 ): string {
-  const target = new URL('auth', domain);
   let msg = '';
   msg += `${domain} wants you to sign in with your account:\n`;
   msg += `\n`;
   if (authParam.statement !== '') {
     msg += `${authParam.statement}\n`;
   }
   msg += '\n';
-  msg += `URI: ${target.href}\n`;
+  // @Yaiba: Should I trust the URI provided by KGW or should I use the domain / create my own?
```

Comment:
@Yaiba, can you advise on this? Should I trust the URI provided by KGW or should I create my own?

Reply:
Since we have checked that the domain matches, KGW will always return `domain + /rpc/v1`, we can trust returned URI.